checkbox-provider/bin/configure

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+# Copyright 2018-2022 Canonical Ltd.
+# All rights reserved.
+#
+# Written by:
+#   Maciej Kisielewski <maciej.kisielewski@canonical.com>
+#   Sylvain Pineau <sylvain.pineau@canonical.com>
+import os
+import re
+import sys
+
+sys.path.append(os.path.expandvars("$SNAP/usr/lib/python3/dist-packages"))
+sitepkgpath = "$SNAP/lib/python3.10/site-packages"
```

Comment:
Similar comment as before - is this the correct python path (should it be 3.12 instead of 3.10)?